controllers/operator_controller.go

```diff
@@ -47,10 +49,33 @@ type OperatorReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.13.1/pkg/reconcile
 func (r *OperatorReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	_ = log.FromContext(ctx)
+	log := log.FromContext(ctx).WithName("reconcile")
+
+	var operator operatorsv1alpha1.Operator
+	if err := r.Get(ctx, req.NamespacedName, &operator); err != nil {
+		log.Error(err, "unable to fetch Operator")
+		return ctrl.Result{}, client.IgnoreNotFound(err)
+	}
+
+	readyCondition := metav1.Condition{
+		Type:               operatorsv1alpha1.TypeReady,
+		Status:             metav1.ConditionFalse,
+		Reason:             "startUp",
```

Comment:
`Reason` should be a `const` in the API package and should be "CamelCased". Clients sometimes use `Reason` in their logic, so this is part of our API.

Also, I know this is probably not meant to stay this way, but the way I would suggest structuring condition modification is:
1. Anytime there's an error during reconciliation, set the conditions relevant to that error.
2. Anytime reconciliation has progressed to the point where you know what a condition will be regardless of any future errors, set that condition.
3. No matter which code path happens, the `Reconcile` function should call `SetStatusCondition` exactly once for every single condition type.

Lastly, what is "startUp" supposed to tell a user about the state of the Operator object? I'd suggest something like reason "NotImplemented" with a message along the lines of "Operator reconciliation is not yet implemented." And then a TODO comment that tells us to remove this once we start building out the implementation.

Reply:
I was kinda going along the lines of "we hit the reconcile function". But it was not meant to stay that way. Will update with TODO, etc.